.github/workflows/main.yml

```diff
@@ -131,15 +131,6 @@ jobs:
           python: '3.11'
           allow_failure: false
 
-        - name: py38-dj42-sqlite-xdist-coverage
-          python: '3.8'
-          allow_failure: false
-
-        # Explicitly test min pytest.
```

Comment:
We should keep one job testing the `pytestmin`. Best to just bump this one to py39.